product_docs/docs/eprs/7/03_installation/installing_jdbc_driver.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Installing a JDBC driver"
+---
+
+Which JDBC driver you use depends on the what database you are using:
+using migrating to or from EDB Postgres Advanced Server, use the EDB JDBC driver. To download the latest driver, see [EDB Connectors](https://enterprisedb.com/software-downloads-postgres#edb-connectors) on the EDB Downloads page. For installation instructions, see [Installing and configuring EDB JDBC Connector](/jdbc_connector/latest/04_installing_and_configuring_the_jdbc_connector/).
+
+- If you're using PostgreSQL, use the PostgreSQL JDBC driver. To download the latest supported driver, see the [JDBC drivers section](https://jdbc.postgresql.org/download/) on the PostgreSQL Downloads page. 
+
+- If you're using Oracle, MySQL, Microsoft SQL Server or Sybase, use the freely available source-specific JDBC driver. 
+  - [Oracle JDBC](https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html)
+  - [MySQL JDBC](https://dev.mysql.com/downloads/connector/j/)
+  - [Microsoft SQL Server JDBC](https://learn.microsoft.com/en-us/sql/connect/jdbc/release-notes-for-the-jdbc-driver?view=sql-server-ver16#102)
+  - [Sybase ASE (jTDS) JDBC](https://sourceforge.net/projects/jtds/files/)
```

Comment:
Replication Server does not support replication with Sybase.  While the jTDS driver can still be used with older versions of SQL Server, our recommendation is to use the Microsoft SQL Server JDBC driver.  As such, I think we should remove the jTDS JDBD driver reference.  If we keep it, I would probably relabel the link to just say "jTDS JDBC" instead of "Sybase ASE (jTDS) JDBC".